crates/subspace-farmer/src/rpc_client/node_rpc_client.rs

```diff
@@ -13,6 +13,10 @@ use subspace_rpc_primitives::{
     FarmerProtocolInfo, RewardSignatureResponse, RewardSigningInfo, SlotInfo, SolutionResponse,
 };
 
+// Defines max_concurrent_requests constant in the node rpc client.
+// It must be set for large plots.
+const WS_PRC_MAX_CONCURRENT_REQUESTS: usize = 1_000_000;
```

Comment:
@shamil-gadelshin I believe this is causing us significant issues now. Instead of having natural request backpressure we're initiating a huge number of requests that time out instead. Do you recall why exactly was this necessary?

Reply:
Maybe because of this one: https://github.com/subspace/subspace/issues/729#issuecomment-1203999225